hooks/msghooks.py

```diff
@@ -246,8 +246,53 @@ def receive_message():
                     + r.json()["message"]
                 )
             return ""
+        
         if re.match(r"^issue", without_ping):
             return reply + ISSUE_HELP
+        
+        if re.match(r"^(update )?prod(uction)?", without_ping):
+            # Create a PR for main -> production
+            # but first check for privleges
+            
+            if message["user_id"] not in STORAGE["privileged"]:
```

Comment:
Maybe this check could be extracted to a helper method

Reply:
I will eventually rework the entire bot to improve the structure and remove duplication. Alternatively, we could PR this separately to fix some similar issues as well.